cheatsheets/JSON_Web_Token_Cheat_Sheet_for_Java.md

```diff
@@ -55,29 +55,29 @@ The tips presented in this article are part of a Java project that was created t
 
 You can find the Java project [here](https://github.com/righettod/poc-jwt), it uses the official [JWT library](https://jwt.io/#libraries).
 
-In the rest of the article, the term **token** refer to the **JSON Web Tokens** (JWT).
+In the rest of the article, the term **token** refers to the **JSON Web Tokens** (JWT).
 
-# Consideration about using JWT
+# Consideration about Using JWT
 
 Even if a JWT token is "easy" to use and allow to expose services (mostly REST style) in a stateless way, it's not the solution that fits for all applications because it comes with some caveats, like for example the question of the storage of the token (tackled in this cheatsheet) and others...
 
-If your application does not need to be fully stateless, you can consider using traditional session system provided by all web frameworks and follow the advice from the dedicated [cheatsheet](Session_Management_Cheat_Sheet.md). However, for stateless applications, when well implemented, it's a good candidate.
+If your application does not need to be fully stateless, you can consider using traditional session system provided by all web frameworks and follow the advice from the dedicated [session management cheat sheet](Session_Management_Cheat_Sheet.md). However, for stateless applications, when well implemented, it's a good candidate.
 
 # Issues
 
-## NONE hashing algorithm
+## `none` Hashing Algorithm
```

Comment:
This looks very strange in a title - I think it'd be better as either the original NONE, or None.